doc/src/vtr/run_vtr_flow.rst

```diff
@@ -309,7 +309,7 @@ Detailed Command-line Options
 .. note::
 
     Universal Hardware Data Model (UHDM) is a complete modeling of the IEEE SystemVerilog Object Model with VPI Interface, Elaborator, Serialization, Visitor and Listener.
-    UHDM is used as a compiled interchange format in between SystemVerilog tools.
+    UHDM is used as a compiled interchange format in between SystemVerilog tools. Typically, files with ``.sv`` (or ``.v``) extensions include UHDM description.
```

Comment:
@vaughnbetz - I have just added a short sentence here to clarify what should be expected for UHDM files. Kindly, let me know if you would like additional modifications.

Reply:
Sure, I will update it now.